### PR TITLE
Use Azure Artifacts for nugetvssprivate

### PR DESCRIPTION
**Pull request by johnterickson**

### CODE REVIEW

File: src/NuGet.Config [hunk 1]
```diff
@@ -3,7 +3,7 @@
   <packageSources>
     <!--To inherit the global NuGet package sources remove the <clear/> line below -->
     <clear />
-    <add key="restsdk" value="https://www.myget.org/F/nugetvssprivate/api/v3/index.json"/>
+    <add key="restsdk" value="https://pkgs.dev.azure.com/mseng/PipelineTools/_packaging/nugetvssprivate/nuget/v3/index.json"/>
```
Comment (bryanmacfarlane) [inline src/NuGet.Config]: Is this public?  Since the agent is OSS, it's a requirement that anyone can clone and build.

Comment (bryanmacfarlane) [reply src/NuGet.Config]: I think it's public since it's a public project but just confirming ...

Comment (johnterickson) [reply src/NuGet.Config]: Yup! Browse here from InPrivate/Incognito: https://dev.azure.com/mseng/PipelineTools/_packaging?_a=feed&feed=nugetvssprivate

Comment (johnterickson) [reply src/NuGet.Config]: PR build is the ultimate test though :)

Comment (bryanmacfarlane) [reply src/NuGet.Config]: cool. looks good to me

Comment (johnterickson) [reply src/NuGet.Config]: (Public feeds are very new so you are spot-on in asking)

Comment (johnterickson) [reply src/NuGet.Config]: @bryanmacfarlane @TingluoHuang I've confirmed that the build publishes to both myget and Azure Artifacts: https://dev.azure.com/mseng/AzureDevOps/_build/results?buildId=10013065

Merge please 😇

**Pull request merged**